algoliasearch-common/src/main/java/com/algolia/search/APIClient.java

```diff
@@ -54,6 +55,20 @@
   }
 
   /**
+   * List all existing indexes
+   *
+   * @return A List of the indices and their metadata
+   * @throws AlgoliaException
+   */
+  public List<Index.Attributes> listIndices(RequestOptions requestOptions) throws AlgoliaException {
```

Comment:
You are right, this PR was only the start of a POC.
So I didn't implement everything yet